server/src/main/java/org/opensearch/index/seqno/ReplicationTracker.java

```diff
@@ -759,10 +787,60 @@ public int hashCode() {
             result = 31 * result + Long.hashCode(globalCheckpoint);
             result = 31 * result + Boolean.hashCode(inSync);
             result = 31 * result + Boolean.hashCode(tracked);
+            result = 31 * result + Objects.hashCode(replicationMode);
             return result;
         }
     }
 
+    /**
+     * The type of replication used for inter-node replication.
+     */
+    public enum ReplicationMode implements Writeable {
+        /**
+         * In this mode, a {@code TransportReplicationAction} is fanned out to underlying concerned shard and is replicated logically.
+         */
+        LOGICAL_REPLICATION(0),
```

Comment:
It has to be `LOGICAL_REPLICATION` which is intended just for translog . Since we wanted to decouple it from translog, we are introducing the `replication mode` which will again confuse . 

Should we use `FanoutMode`  instead of  `LOGICAL_REPLICATION` ?

Reply:
@gbbafna  I have renamed the same. Let me know if that is okay.